.github/workflows/continous-integration-os.yml

```diff
@@ -152,7 +159,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [stable]
+        rust: [stable, beta]
```

Comment:
The fake activation heights test results shouldn't depend on the OS and Rust release.

Can we remove this job entirely, and just use "Test with fake activation heights" (Docker) instead?

Reply:
I'm just going to go ahead and make this change, so we can merge.

I'm happy to revert later if you'd like.